core/src/main/java/org/opensearch/sql/data/model/ExprValue.java

```diff
@@ -133,13 +133,9 @@ default LocalDate dateValue() {
         "invalid to get dateValue from value of type " + type());
   }
 
-  /** Get datetime value. */
-  default LocalDateTime datetimeValue() {
-    throw new ExpressionEvaluationException(
-        "invalid to get datetimeValue from value of type " + type());
-  }
-
-  /** Get interval value. */
+  /**
+   * Get interval value.
```

Comment:
Collapse the javadoc into one line to satisfy spotless

Reply:
fixed in https://github.com/Bit-Quill/opensearch-project-sql/commit/f6edc6be62abbe3a3974665e8d51c515584234e0